ibm_catalog.json

```diff
@@ -0,0 +1,145 @@
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-ocp-ai/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ocp-ai/main/images/rh_ocp_ai.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "AI-Enhanced Red Hat OpenShift Deployment",
+            "description": "Automates the provisioning of a fully configured Red Hat OpenShift on VPC cluster with integrated AI capabilities on IBM Cloud. This includes optimized configurations for machine learning frameworks and simplified deployment of AI-driven applications. Refer the documentation to better understand the service and its capabilities: https://cloud.ibm.com/docs/openshift?topic=openshift-ai-addon-about"
+          }
+        ],
+        "flavors": [
+          {
+            "label": "Openshift AI",
+            "name": "openshift-ai",
+            "install_type": "fullstack",
+            "working_directory": "solutions/ocp-ai",
+            "architecture": {
+              "descriptions": "This architecture deploys a single-zone Red Hat OpenShift on VPC cluster enhanced with an AI addon, accelerating machine learning and AI workload deployments. It outlines the necessary infrastructure components for AI-powered applications and provides a streamlined setup for data science and AI development.",
+              "features": [
+                {
+                  "title": "Create Virtual Private Cloud (VPC)",
+                  "description": "No"
+                },
+                {
+                  "title": "Create Red Hat OpenShift Cluster",
+                  "description": "Yes"
+                },
+                {
+                  "title": "Use Existing VPC instance",
+                  "description": "No"
+                },
+                {
+                  "title": "New Resource Group Creation",
+                  "description": "No"
+                }
+              ],
+              "diagrams": [
+                {
+                  "diagram": {
+                    "caption": "Red Hat OpenShift cluster with AI addon for IBM Cloud.",
+                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ocp-ai/blob/main/reference-architecture/deployable-architecture-ocp-cluster-qs.svg",
+                    "type": "image/svg+xml"
+                  },
+                  "description": "This architecture illustrates the deployment of a single-zone Red Hat OpenShift cluster integrated with an AI addon, showcasing the key components and their interactions for AI workload support."
```

Comment:
Its not only `single-zone`...so single-zone can be removed.

Reply:
done